bots/psuteam7bot/crusader.js

```diff
@@ -128,11 +119,15 @@ crusader.takeAttackerAction = (self) => {
     }
     else if(self.squadSize === 0)
     {
+        if(self.fuel < 100)
```

Comment:
Interesting add. Was it having any beneficial effect from replay testing? Feel like this could be good and bad in different scenarios

Reply:
I forgot about this change, but it was helping a bit with having enough fuel for signal the next round, especially on replays with 2 castles. For 3 castles replays, crusader production and crusader moving to rally still cause fuel scarcity.